utils/payloads.py

```diff
@@ -556,7 +556,7 @@ def random_uuid():
 
 
 IS_EDGE = os.environ.get("IS_EDGE", False)
-USE_RANDOMNESS = os.environ.get("USE_RANDOMENESS", True)
+USE_RANDOMNESS = os.environ.get("USE_RANDOMNESS", "True").lower() in ("true", "True")
```

Comment:
If you're converting to lowercase, you don't need to check against both values
```suggestion
USE_RANDOMNESS = os.environ.get("USE_RANDOMNESS", "True").lower() == "true"
```